### DISCUSSION TITLE
可以将‘定理环境’中的证明字体改成斜体吗（默认是粗体好像）

### DISCUSSION THREAD

**Discussion by Juelianqvq** [stone-zeng/fduthesis#246] [Q&A] [answered]
可以将‘定理环境’中的“证明”字体改成斜体的“证明.
![屏幕截图 2022-03-01 230112](https://user-images.githubusercontent.com/42963620/156193087-a78bd0f7-032b-463d-b216-b86000282cc0.png)
![屏幕截图 2022-03-01 230132](https://user-images.githubusercontent.com/42963620/156193091-913febb0-cd49-40b0-87af-cc6bcbf75068.png)
”吗

**Comment by stone-zeng** [comment, marked as answer]
```tex
\newtheorem*[header-font=\itshape]{myproof}{证明}
```

<img width="957" alt="Screen Shot 2022-03-02 at 02 24 37" src="https://user-images.githubusercontent.com/18233088/156226890-1cbf46d1-78f8-41dd-a506-8dfc0b7ef397.png">

**Reply by Juelianqvq** [reply]
非常感谢！辛苦啦
